docs/aetest/loop.rst

```diff
@@ -22,102 +21,104 @@ Looping Sections
+    In other words, ``subsection``, ``Testcase`` and ``test`` sections are the
     only loopable sections.
 
-
 Defining Loops
 --------------
 
-Sections are marked for looping when they are decorated with ``@loop``, and its
-looping parameters provided as decorator arguments. During runtime, when 
+Sections are marked for looping when decorated with ``@loop``, and its
+looping parameters are provided as decorator arguments. During runtime, when 
 ``aetest`` infrastructure detects looped section code, their corresponding 
-section object is then instantiated once for each of its iterations.
+section object is then instantiated once for each iteration.
 
 .. code-block:: python
 
     # Example
     # -------
     #
-    #   defining loops on sections
+    #   Defining loops on sections
 
     from pyats import aetest
 
-    # defining a common setup section
-    # contains a subsection that is looped twice.
+    # Defining a common setup section
+    # The CommonSetup section contains a 
+    # subsection that is looped twice.
     class CommonSetup(aetest.CommonSetup):
 
-        # defining a subsection
-        # this subsection is marked to be looped twice
-        # the first time having a uid of "subsection_one", and 
+        # Defining a subsection
+        # ------------------------
+        # This subsection is marked to be looped twice.
+        # The first time having a uid of "subsection_one", and 
         # the second time having a uid of "subsection_two"
         @aetest.loop(uids=['subsection_one', 'subsection_two'])
         @aetest.subsection
         def looped_subsection(self):
             pass
 
-    # defining a testcase that loops
-    # this testcase also contains a test section that is looped twice
+    # Defining a test case that loops
+    # ----------------------------------
+    # This test case also contains a test section that is looped twice
     @aetest.loop(uids=['testcase_one', 'testcase_two'])
     class Testcase(aetest.Testcase):
 
-        # setup section of this testcase is run once
-        # every time the testcase is looped.
+        # Setup section of this test case is run once
+        # every time the test case is looped.
         @aetest.setup
         def setup(self):
             pass
 
-        # looped test section
-        # both iterations are run per testcase iteration
+        # Looped test section
+        # both iterations are run per test case iteration
         @aetest.loop(uids=['test_one', 'test_two'])
         @aetest.test
         def test(self):
             pass
 
-        # cleanup section of this testcase is run once
-        # every time the testcase is looped.
+        # Cleanup section of this test case is run once
+        # every time the test case is looped.
         @aetest.cleanup
         def cleanup(self):
             pass
 
-    # this testscript's resulting sections would look like this below
+    # This test script's resulting sections would look like this below
```

Comment:
It's not clear if this is referring to a code object type so it's probably safe to leave this as two words.